IdentityCore/src/oauth2/account/MSIDAccountType.m

```diff
@@ -53,7 +56,8 @@ + (MSIDAccountType)accountTypeFromString:(NSString *)type
         
         sAccountTypes = @{@"aad": @(MSIDAccountTypeAADV1),
                           @"msa": @(MSIDAccountTypeMSA),
-                          @"mssts": @(MSIDAccountTypeMSSTS)};
+                          @"mssts": @(MSIDAccountTypeMSSTS),
+			  @"adfs": @(MSIDAccountTypeADFS)};
```

Comment:
nit: fix the identation 
```suggestion
                          @"adfs": @(MSIDAccountTypeADFS)};
```

Reply:
Will fix indentation in a follow up PR since change was already released in hotfix release